src/sagemaker/tensorflow/README.rst

```diff
@@ -558,14 +557,13 @@ For more information on training and evaluation process, see `tf.estimator.train
 
 For more information on fit, see `SageMaker Python SDK Overview <#sagemaker-python-sdk-overview>`_.
 
-TensorFlow serving models
+TensorFlow Serving models
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 After your training job is complete in SageMaker and the ``fit`` call ends, the training job
-will generate a `TensorFlow serving <https://www.tensorflow.org/serving/serving_basic>`_
-model ready for deployment. Your TensorFlow serving model will be available in the S3 location
-``output_path`` that you specified when you created your `sagemaker.tensorflow.TensorFlow`
-estimator.
+will generate a `TensorFlow SavedModel <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md>`_
+bundle ready for deployment. Your model will be available in S3 at the ``output_path`` location
+that you specified when you created your `sagemaker.tensorflow.TensorFlow` estimator.
```

Comment:
add another set of backticks around `sagemaker.tensorflow.TensorFlow`

Reply:
ok